### PR TITLE
os: Mac package

### PR DESCRIPTION
**Pull request by afwerar**

### CODE REVIEW

**Review by sangshuduo — CHANGES_REQUESTED** [review]
Could we exclude the binary file TDengine.app.tar.gz from the GitHub repo?

File: packaging/tools/make_install.sh [hunk 1]
```diff
@@ -209,18 +194,20 @@ function install_bin() {
...
-    [ -x ${install_main_dir}/bin/taosx ] || [ -x ${install_main_2_dir}/bin/taosx ] && ${csudo}ln -s ${install_main_dir}/bin/taosx ${bin_link_dir}/taosx || ln -s ${install_main_2_dir}/bin/taosx ${bin_link_dir}/taosx || :
+    [ -x ${install_main_dir}/bin/${clientName} ] && ${csudo}ln -s ${install_main_dir}/bin/${clientName} ${bin_link_dir}/${clientName} || :
+    [ -x ${install_main_dir}/bin/${serverName} ] && ${csudo}ln -s ${install_main_dir}/bin/${serverName} ${bin_link_dir}/${serverName} || :
+    [ -x ${install_main_dir}/bin/taosadapter ] && ${csudo}ln -s ${install_main_dir}/bin/taosadapter ${bin_link_dir}/taosadapter || :
+    [ -x ${install_main_dir}/bin/udfd ] && ${csudo}ln -s ${install_main_dir}/bin/udfd ${bin_link_dir}/udfd || :
+    [ -x ${install_main_dir}/bin/taosdump ] && ${csudo}ln -s ${install_main_dir}/bin/taosdump ${bin_link_dir}/taosdump || :
+    [ -x ${install_main_dir}/bin/taosdemo ] && ${csudo}ln -s ${install_main_dir}/bin/taosdemo ${bin_link_dir}/taosdemo || :
```
Comment (sangshuduo) [inline packaging/tools/make_install.sh]: not sure if taosBenchmark is placed right here

**Review by sangshuduo — COMMENTED** [review]
Interested that the Icon file is empty.

**Review by sangshuduo — APPROVED** [review]
LGTM for overall commiting. Please make sure taosBenchmark will be installed correctly.

File: packaging/tools/make_install.sh (hunk 1 above)
Comment (afwerar) [reply packaging/tools/make_install.sh]: I fixed it

**Pull request merged**